WordPress/wporg-main-2022
# Adjust heading sizes and margin

Fixes #246 

This PR 
* Removes `has-heading-2-font-size` for `<h1>`. Except for the `<h1>` in the download subpage. Reason see below.
* Adds `margin-top:var(--wp--preset--spacing--40)` for `<h2>` and `margin-top:var(--wp--preset--spacing--30)` for `<h3>`. And the `<h2>` and `<h3>` that have their font size overridden weren't added with it, as the layout would go off consequently.




## Screenshots



### Philosophy
| Before | After |
|--------|-------|
| ![Screen Shot 2023-04-11 at 10 54 26 AM](https://user-images.githubusercontent.com/18050944/231043835-aa637b27-e5f8-4668-b98f-3dcda9050934.png) | ![Screen Shot 2023-04-11 at 10 53 00 AM](https://user-images.githubusercontent.com/18050944/231043607-10dcc7ae-4e6d-4dd3-8c31-0e8d66a4e9ef.png) |

### Graphics & Logos
| Before | After |
|--------|-------|
| ![Screen Shot 2023-04-11 at 11 01 08 AM](https://user-images.githubusercontent.com/18050944/231045183-761d545b-209f-49b9-900f-657b6b7fd73c.png) | ![Screen Shot 2023-04-11 at 11 02 28 AM](https://user-images.githubusercontent.com/18050944/231045312-4bab6c7f-e27b-4789-a14f-0acda8a60b56.png) |

### Security
| Before | After |
|--------|-------|
| ![Screen Shot 2023-04-11 at 11 01 15 AM](https://user-images.githubusercontent.com/18050944/231045355-20df24ee-6f2a-4dcf-8ba4-0a7b98e718f5.png) | ![Screen Shot 2023-04-11 at 11 02 47 AM](https://user-images.githubusercontent.com/18050944/231045987-c5b542c8-695e-45b2-af04-326825aa4d56.png) |
| ![Screen Shot 2023-04-11 at 11 01 27 AM](https://user-images.githubusercontent.com/18050944/231045385-339d2a1e-e823-4edc-b8d2-26d229c3042e.png) | ![Screen Shot 2023-04-11 at 11 02 54 AM](https://user-images.githubusercontent.com/18050944/231045459-6ad3f0e7-cb56-40b9-87b1-81d83236f481.png) |

### Download Subpage

The adjustments made to the font size of the `<h1>` on the download subpages were reverted as the layout was off, even with `nowrap` added. 

| Before | After | Nowrap |
|--------|--------|-------|
| ![Screen Shot 2023-04-11 at 11 13 26 AM](https://user-images.githubusercontent.com/18050944/231046778-05a79615-79de-48b3-92c5-e67d005ae69b.png) | ![Screen Shot 2023-04-11 at 10 49 52 AM](https://user-images.githubusercontent.com/18050944/231046489-d0034ead-22ef-4551-ba33-52a1b006c358.png) | ![Screen Shot 2023-04-11 at 10 49 58 AM](https://user-images.githubusercontent.com/18050944/231046505-56cc5152-a942-406e-b257-67d1a6b60ef2.png) |

### How to test the changes in this Pull Request:

1. Go to `/about/*`.
2. Check if `<h1>` is of `font-size` `70px` and `<h2>, <h3>` is of `margin-top: 40px, margin-top: 30px` respectively.
